packages/camera/camera/ios/Classes/CameraPlugin.m

```diff
@@ -158,6 +158,8 @@ - (void)handleMethodCallAsync:(FlutterMethodCall *)call
   } else if ([@"stopImageStream" isEqualToString:call.method]) {
     [_camera stopImageStream];
     [result sendSuccess];
+  } else if ([@"receivedImageStreamData" isEqualToString:call.method]) {
+    _camera.streamingPendingFrames--;
```

Comment:
This should be an internal detail; the public API for FLTCam should be a method that acknowledges receipt, not a counter. Incrementing and decrementing a variable shouldn't be split between two classes.